### ISSUE TITLE
[BUG] Error compiling cython bindings during build

### ISSUE THREAD

**Issue by randerzander** [rapidsai/cuspatial#22]
Following the build instructions in README.md I get the following error snippet:
```
(rapids) root@1bdf09244fab:/rapids/cuspatial/python/cuspatial# python setup.py build_ext --inplace           
Compiling cuspatial/bindings/query.pyx because it changed.                                                       
Compiling cuspatial/bindings/soa_readers.pyx because it changed.                                                 
Compiling cuspatial/bindings/spatial.pyx because it changed.                                                     
Compiling cuspatial/bindings/trajectory.pyx because it changed.                             
[1/4] Cythonizing cuspatial/bindings/query.pyx                                                        
                                                                                                  
Error compiling Cython file:
------------------------------------------------------------
...
# cython: profile=False
# distutils: language = c++
# cython: embedsignature = True
# cython: language_level = 3

from cudf.bindings.cudf_cpp cimport *
^
------------------------------------------------------------

cuspatial/bindings/query.pxd:8:0: 'cudf/bindings/cudf_cpp.pxd' not found

Error compiling Cython file:
------------------------------------------------------------
...
from cudf.bindings.cudf_cpp cimport *
from libcpp.pair cimport pair                                                                                                                  
cdef extern from "query.hpp" namespace "cuspatial" nogil:                                                                                     

    cdef pair[gdf_column, gdf_column] spatial_window_points(
                       ^
------------------------------------------------------------

cuspatial/bindings/query.pxd:13:24: unknown type in template argument
```

Omitted:
Many repeated failured about ```cuspatial/bindings/query.pxd:14:14: 'gdf_scalar' is not a type identifier``` and ```cuspatial/bindings/query.pxd:18:14: 'gdf_column' is not a type identifier``` from different .pxd files.

**Comment by randerzander** [comment]
Perhaps this is fixed by #18 ?

**Comment by thomcom** [comment]
It should be fixed by #18. If you still have this problem after updating to #18 you should blow away your python bindings directory with `cd $CUSPATIAL_HOME/python/cuspatial/; rm -rf build`

**Comment by thomcom** [comment]
Arg! I messed something up in a merge. This was fixed by #23

**Comment by randerzander** [comment]
My build problems resolved after #23, thanks @thomcom!
